test/e2e/webdriver/driver.js

```diff
@@ -640,7 +640,6 @@ function logBrowserError(failOnConsoleError, errorMessage) {
 
   if (failOnConsoleError) {
     console.error('-----failOnConsoleError is true------\n');
-    throw new Error(errorMessage);
```

Comment:
If this ends up being the fix, it can be configured on the specific test instead:

```
await withFixtures({
  ...,
  failOnConsoleError: false,
}
```

Reply:
Yeah, we definitely don't want to remove this line. This setting is supposed to make tests fail when we see an error message.

Reply:
yes, i figured out a mock now and can bring it back, see [567583](https://github.com/MetaMask/metamask-extension/pull/22548/commits/567583ece63b2f4538040c54e669a308d21bbf63)